README.md

```diff
@@ -1,10 +1,24 @@
 # Lora Based BraveSensor Firmware
 
+# Project Overview
+
+## Setup
+Follow this [guide](https://docs.rakwireless.com/Product-Categories/WisDuo/RAK3272S-Breakout-Board/Quickstart/#rak3272s-breakout-board-as-a-stand-alone-device-using-rui3) on how to setup. The LoraBraveSensor directory can subsitute the Arduino Serial demo, as a guide on how to compile and upload our custom firmware. After going throught the entire process, I recommend using vscode's Arduino extension for easier file management. 
+
+## Secrets File
```

Comment:
Can this section also say where the `secrets.h` file needs to live in order for this to work?

Reply:
Correct, I've swapped the ordering.

Reply:
See below comment about 'can' vs 'must'